app/views/events/index.html.erb

```diff
@@ -51,6 +51,9 @@
       <div class="col-12">
         <%= workshops_link(params[:type])%>
       </div>
+      <div class="col-12">
```

Comment:
These should appear horizontally, with a pipe separator. Responsive views can wrap links if necessary.